readthedocs/core/apps.py

```diff
@@ -0,0 +1,12 @@
+from django.apps import AppConfig
+
+
+class CoreAppConfig(AppConfig):
+    name = 'readthedocs.core'
+    verbose_name = 'Core'
+
+    def ready(self):
+        if hasattr(self, 'already_run'):
+            return
+        self.already_run = True
```

Comment:
Ah, here's more info, there's a note that says testing applications could execute `ready` more than once:
https://docs.djangoproject.com/en/1.8/ref/applications/#django.apps.AppConfig.ready

If we do need this, `if hasattr(...)` should be `getattr(self, 'already_run', False)`